src/test/java/org/eclipse/yasson/customization/ImplementationClassTest.java

```diff
@@ -36,23 +37,24 @@ public void testAnnotatedImplementation() {
         String expected = "{\"animal\":{\"dogProperty\":\"Bulldog\"}}";
         String json = jsonb.toJson(pojo);
 
-        Assert.assertEquals(expected, json);
+        assertEquals(expected, json);
         ImplementationClassPojo result = jsonb.fromJson(expected, ImplementationClassPojo.class);
-        Assert.assertTrue(result.getAnimal() instanceof Dog);
-        Assert.assertEquals("Bulldog", ((Dog)result.getAnimal()).getDogProperty());
+        assertTrue(result.getAnimal() instanceof Dog);
+        assertEquals("Bulldog", ((Dog)result.getAnimal()).getDogProperty());
     }
 
     @Test
     public void testJsonbConfigUserImplementation() {
         HashMap<Class, Class> userMapping = new HashMap<>();
         userMapping.put(Animal.class, Dog.class);
-        Jsonb jsonb = JsonbBuilder.create(new JsonbConfig().setProperty(YassonProperties.USER_TYPE_MAPPING, userMapping));
+        Jsonb jsonb = JsonbBuilder.create(new JsonbConfig().setProperty("jsonb.user-type-mapping", userMapping));
```

Comment:
I prefer using variables instead of hardcoded strings in the tests, but I see what you are getting at -- you want to hardcode the values so it will be obvious if we ever change the property names for some reason.

Instead, lets just add a single test that verifies all of the `YasssonProperties` constants as their respective hardcoded strings, which will achieve the same result of making sure we never accidentally change the property names.

Reply:
That is something I can agree on.